service/worker/diagnostics/activities.go

```diff
@@ -71,7 +72,7 @@ type rootCauseTimeoutsParams struct {
 }
 
 func (w *dw) rootCauseTimeouts(ctx context.Context, info rootCauseTimeoutsParams) ([]invariants.InvariantRootCauseResult, error) {
-	timeoutInvariant := invariants.NewTimeout(invariants.NewTimeoutParams{
+	timeoutInvariant := timeouts.NewTimeout(timeouts.NewTimeoutParams{
```

Comment:
nit: I think that the refactoring changes the organization of the folders and all types of invariants will be placed in `invariants/<invariant type>` folder? The package name `timeouts` already has a name of the type, so I think it would be more correct to have `timeoutInvariant := timeout.NewInvariant` in this case, to avoid the duplication.